xendit_pos/static/src/js/payment_xendit_pos.js

```diff
@@ -17,15 +17,22 @@ odoo.define('xendit_pos.payment', function (require) {
             return this._xendit_pay();
         },
 
+        get_selected_payment: function () {
```

Comment:
@andykim random question, why some function name has `_` prepended and some doesn't? Is there a specific style guide for the naming convention with `_`?

Reply:
@ikanyu I think it's because when we init the code from a default Odoo addon. I don't like it too. :)